packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -763,6 +779,8 @@ const { HeadTags, CitationTags, PostsPagePostHeader, LWPostsPageHeader, PostsPag
     }
   }, [fullPost, hashCommentId, isDebateResponseLink, linkedCommentId, showHashCommentFallback])
 
+  const splashPageHeader = fullPost && <BestOfLWPostsPageSplashImage post={fullPost} />
```

Comment:
seems like this is no longer guarded to posts that would have a splash page header? Are we dealing with that by not rendering most stuff if there's no URL? But I guess we are adding a bunch of nodes to the DOM?